client/verta/tests/conftest.py

```diff
@@ -520,13 +531,43 @@ def predict(self, x):
     return ep
 
 
-class Workspace:
-    def __init__(self, name):
-        self.name = name
-
 @pytest.fixture
 def workspace(client, created_entities):
-    return Workspace(client._conn.get_default_workspace())
+    return create_workspace(client, created_entities, [
+    RoleV2_pb2.RoleResourceActions(resource_type=RoleV2_pb2.ResourceTypeV2.ENDPOINT,
+                                   allowed_actions=[RoleV2_pb2.ActionTypeV2.READ]),
+    RoleV2_pb2.RoleResourceActions(resource_type=RoleV2_pb2.ResourceTypeV2.REGISTERED_MODEL,
+                                   allowed_actions=[RoleV2_pb2.ActionTypeV2.READ,
+                                                    RoleV2_pb2.ActionTypeV2.UPDATE]),
+])
```

Comment:
The indentation seems a bit strange to me, here. I would recommend trying this:

```suggestion
    return create_workspace(
        client,
        created_entities,
        [
            RoleV2_pb2.RoleResourceActions(
                resource_type=RoleV2_pb2.ResourceTypeV2.ENDPOINT,
                allowed_actions=[RoleV2_pb2.ActionTypeV2.READ],
            ),
            RoleV2_pb2.RoleResourceActions(
                resource_type=RoleV2_pb2.ResourceTypeV2.REGISTERED_MODEL,
                allowed_actions=[RoleV2_pb2.ActionTypeV2.READ,
                                 RoleV2_pb2.ActionTypeV2.UPDATE],
            ),
        ],
    )
```

(It's a bit more conventional this way, even though it's more lines)